### PR TITLE
Add support for Nightwatch v3.

### PR DESCRIPTION
**Pull request by garg3133**
Nightwatch v3 release: https://github.com/nightwatchjs/nightwatch/releases/tag/v3.0.1
Updated website: https://nightwatchjs.org/

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,11 +29,10 @@
     "@percy/sdk-utils": "^1.0.0"
   },
   "peerDependencies": {
-    "nightwatch": ">=1 || >=2"
+    "nightwatch": ">=1 || >=2 || >=3"
```
Comment (itsjwala) [inline package.json:32]: bump `@percy/cli` as well to `1.26.1`

**Comment by garg3133** [comment]
@itsjwala Can you please approve the CI run?

**Comment by itsjwala** [comment]
@garg3133 done, push a empty commit

**Comment by itsjwala** [comment]
hmm, I updated some GitHub settings to avoid approval of actions run, can you push an empty commit again?

**Comment by itsjwala** [comment]
pushed a commit in master, the `pull_request`. trigger was missing which was causing issues. 

had to close and reopen the PR to retrigger the checks

**Review by itsjwala — APPROVED** [review]
LGTM, please update the description or maybe attach link to nightwatch repo/ doc if you require backlinks

**Pull request merged**